gatsby/src/pages/security-disclosure-policy.js

```diff
@@ -9,6 +9,13 @@ import config from '../../config'
 
 const HallOfFame = () => {
     const hallOfFameEntries = [
+        {
+            date:'2020-09-20',
+            who: 'Denis Kasak',
+            profile: 'https://github.com/dkasak',
+            description: 'HTML injection in reCAPTCHA endpoint leads to reflected XSS',
```

Comment:
It'd be nice if this could (a) say when it was fixed (Synapse 1.21.0 I think?) and (b) maybe give the CVE #, since we have one? Also the description isn't very clear.

```suggestion
            description: 'HTML injection in login fallback endpoints could be used for a Cross-site-scripting attack (<a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-26891">CVE-2020-26891</a>). Fixed in Synapse 1.21.0.',
```